### ISSUE TITLE
Phototour dataset links are broken.

### ISSUE THREAD

**Issue by aytackanaci** [pytorch/vision#293]
Hi there,
I was trying triplet code on the this fork https://github.com/edgarriba/examples
But dataset link are broken? @edgarriba can you prvoide updated links?

thanks,
Aytac

**Comment by alykhantejani** [comment]
Hi @aytackanaci,

The dataset is hosted as `iis.ee.ic.ac.uk` under @vbalnt's homepage. Currently `iis.ee.ic.ac.uk` seems to be down, hence the data cannot be pulled. I assume this will be fixed shortly, if not I think we can pull the data from [here](http://phototour.cs.washington.edu/patches/default.htm) instead.

**Comment by alykhantejani** [comment]
Website is still down, let's wait until mid-week to see if they resolve it. The site `ic.ac.uk` is down so I'm sure someone there will see this soon.

**Comment by edgarriba** [comment]
@aytackanaci @alykhantejani let me ping directly them

**Comment by alykhantejani** [comment]
@edgarriba @aytackanaci I checked with Vassileios (@vbalnt) yesterday. He is in the process of migrating the data of the IC servers (as they are down).  Once he has uploaded them somewhere else, we will change the links in the dataset.

**Comment by vbalnt** [comment]
Hi @alykhantejani  @edgarriba @aytackanaci 

Sorry for this, some servers at Imperial College had issues. Please find the new links below

```
http://icvl.ee.ic.ac.uk/vbalnt/liberty.zip
http://icvl.ee.ic.ac.uk/vbalnt/notredame.zip
http://icvl.ee.ic.ac.uk/vbalnt/yosemite.zip
```

**Comment by alykhantejani** [comment]
Thanks @vbalnt - I'll update the links in the dataset now

**Comment by alykhantejani** [comment]
This is fixed in master (via #301)
